pyspeckit/spectrum/classes.py

```diff
@@ -193,6 +193,8 @@ def __init__(self, filename=None, filetype=None, xarr=None, data=None,
 
         if unit is not None:
             self._unit = unit
+        # else:
+        #     self._unit = u.dimensionless_unscaled
```

Comment:
OK, now I understand.  I'm not entirely sure what to do here off the top of my head, but we should fix the calls to `_unit` to deal with #48 - we should not be calling that attribute.